src/metrics/mendel_metrics.nit

```diff
@@ -1,7 +1,5 @@
 # This file is part of NIT ( http://www.nitlanguage.org ).
 #
-# Copyright 2014 Alexandre Terrasa <alexandre@moz-code.org>
-#
```

Comment:
Please read my response to your previous commit. And you can't remove some else copyright without his approval.